app/controlplane/plugins/sdk/v1/helpers.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 type renderer struct {
-	render func(t table.Writer) string
-	format string
+	render  func(t table.Writer) string
+	format  string
```

Comment:
After offline conversation, should we remove it?

Reply:
Good catch, removed.

Thanks!